addons/l10n_vn/data/l10n_vn_chart_data.xml

```diff
@@ -2,147 +2,225 @@
+        <field name="name">VN - Chart of Accounts - C200</field>
+        <field name="code_digits">5</field>
         <field name="currency_id" ref="base.VND"/>
         <field name="bank_account_code_prefix">112</field>
         <field name="cash_account_code_prefix">111</field>
         <field name="transfer_account_code_prefix">113</field>
+        <field name="transfer_account_id" ref="transfer_account_id"/>
+        <field name="spoken_languages" eval="'vi_VN'"/>
+    </record>
+
+    <record id="transfer_account_id" model="account.account.template">
+        <field name="chart_template_id" ref="vn_template"/>
     </record>
 
     <!-- Account Templates -->
 
-    <!-- Loại 1 -->
+    <!-- Type 1: Current Assets -->
+    <record id="chart1112" model="account.account.template">
+        <field name="code">1112</field>
+        <field name="reconcile" eval="False"/>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
+        <field name="chart_template_id" ref="l10n_vn.vn_template"/>
+        <field name="name">Foreign currencies</field>
+        <field name="tag_ids" eval="[(6,0,[ref('account_tag_111')])]"/>
+    </record>
+    <record id="chart1113" model="account.account.template">
+        <field name="code">1113</field>
+        <field name="reconcile" eval="False"/>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
+        <field name="chart_template_id" ref="l10n_vn.vn_template"/>
+        <field name="name">Monetary Gold</field>
+        <field name="tag_ids" eval="[(6,0,[ref('account_tag_111')])]"/>
+    </record>
+
+    <record id="chart1122" model="account.account.template">
+        <field name="code">1122</field>
+        <field name="reconcile" eval="False"/>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
+        <field name="chart_template_id" ref="l10n_vn.vn_template"/>
+        <field name="name">Foreign currencies</field>
+        <field name="tag_ids" eval="[(6,0,[ref('account_tag_112')])]"/>
+    </record>
+    <record id="chart1123" model="account.account.template">
+        <field name="code">1123</field>
+        <field name="reconcile" eval="False"/>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
+        <field name="chart_template_id" ref="l10n_vn.vn_template"/>
+        <field name="name">Monetary Gold</field>
+        <field name="tag_ids" eval="[(6,0,[ref('account_tag_112')])]"/>
+    </record>
+
+    <record id="chart1131" model="account.account.template">
+        <field name="name">Vietnam Dong</field>
+        <field name="code">1131</field>
+        <field name="reconcile" eval="True"/>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
+        <field name="tag_ids" eval="[(6,0,[ref('account_tag_113')])]"/>
+    </record>
+    <record id="chart1132" model="account.account.template">
+        <field name="name">Foreign Currencies</field>
```

Comment:
Use a full name to differentiate from the other accounts  `<field name="name">Cash in transit - foreign currencies</field>`